src/video2x-qt6_ja_JP.ts

```diff
@@ -1,742 +1,743 @@
+      <source>Remove Selected</source>
+      <translation>選択項目を削除</translation>
     </message>
     <message>
-        <location filename="taskconfigdialog.ui" line="1098"/>
-        <source>Apply</source>
-        <translation>適用</translation>
+      <location filename="taskconfigdialog.ui" line="1098" />
+      <source>Apply</source>
+      <translation>適用</translation>
     </message>
     <message>
-        <location filename="taskconfigdialog.ui" line="1105"/>
-        <source>Cancel</source>
-        <oldsource>Canacel</oldsource>
-        <translation>キャンセル</translation>
+      <location filename="taskconfigdialog.ui" line="1105" />
+      <source>Cancel</source>
+      <oldsource>Canacel</oldsource>
+      <translation>キャンセル</translation>
     </message>
     <message>
-        <location filename="taskconfigdialog.cpp" line="47"/>
-        <source>Option</source>
-        <translation>オプション</translation>
+      <location filename="taskconfigdialog.cpp" line="47" />
+      <source>Option</source>
+      <translation>オプション</translation>
     </message>
     <message>
-        <location filename="taskconfigdialog.cpp" line="47"/>
-        <source>Value</source>
-        <translation>値</translation>
+      <location filename="taskconfigdialog.cpp" line="47" />
+      <source>Value</source>
+      <translation>値</translation>
     </message>
     <message>
-        <location filename="taskconfigdialog.cpp" line="211"/>
-        <source>Error</source>
-        <translation>エラー</translation>
+      <location filename="taskconfigdialog.cpp" line="211" />
+      <source>Error</source>
+      <translation>エラー</translation>
     </message>
     <message>
-        <location filename="taskconfigdialog.cpp" line="274"/>
-        <source>No GPUs with Vulkan support found!</source>
-        <translation>Vulkan をサポートする GPU が見つかりませんでした！</translation>
+      <location filename="taskconfigdialog.cpp" line="274" />
+      <source>No GPUs with Vulkan support found!</source>
+      <translation>Vulkan をサポートする GPU が見つかりませんでした！</translation>
     </message>
     <message>
-        <location filename="taskconfigdialog.cpp" line="752"/>
-        <source>Invalid noise level for Real-CUGAN model &apos;models-pro&apos;.
+      <location filename="taskconfigdialog.cpp" line="752" />
+      <source>Invalid noise level for Real-CUGAN model 'models-pro'.
 Allowed values are -1, 0, and 3</source>
-        <translation>Real-CUGAN モデル &apos;models-pro&apos; のノイズレベルが無効です。  
-許容される値は -1、0、および 3 です</translation>
+      <translation>Real-CUGAN モデル「models-pro」のノイズレベルが無効です。  
+許容される値は -1、0、および 3 です。</translation>
     </message>
     <message>
-        <location filename="taskconfigdialog.cpp" line="759"/>
-        <source>Invalid codec value: &apos;</source>
-        <translation>無効な codec の値: &apos;</translation>
+      <location filename="taskconfigdialog.cpp" line="759" />
+      <source>Invalid codec value: '</source>
+      <translation>無効なコーデックの値: '</translation>
```

Comment:
I kinda want to keep `codec` as is here so it's consistent with the rest of the messages. `codec`, and the `pix_fmt` and other names here refer to FFmpeg option names.